torchvision/prototype/features/_encoded.py

```diff
@@ -46,11 +46,6 @@ def image_size(self) -> Tuple[int, int]:
 
         return self._image_size
 
-    def decode(self) -> Image:
```

Comment:
Mypy was complaining with:
```
torchvision/prototype/features/_encoded.py:52: error: Returning Any from function declared to return "Image"  [no-any-return]
            return self._F.decode_image_with_pil(self)
```

It's unclear why it couldn't see that the return type is indeed `Image` but given it's not used anywhere in the codebase and it's for debugging only, I'm removing it. Feel free to add back on the future,